From 36207ab1e0963fbbe7941d1b18c5e499e02e682b Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Fri, 9 Jul 2021 15:09:09 +0200
Subject: [PATCH] Solus tweaks to kdeglobals defaults

- Set SolusDark as default Darktheme
- Set SolusLight as default Lighttheme
- Set single click to select files instead of selecting them
---
 kcms/landingpage/landingpage_kdeglobalssettings.kcfg | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/kcms/landingpage/landingpage_kdeglobalssettings.kcfg b/kcms/landingpage/landingpage_kdeglobalssettings.kcfg
index 6bcce1ebd..81d0565b8 100644
--- a/kcms/landingpage/landingpage_kdeglobalssettings.kcfg
+++ b/kcms/landingpage/landingpage_kdeglobalssettings.kcfg
@@ -7,25 +7,25 @@
   <group name="General">
     <entry name="colorScheme" key="ColorScheme" type="String">
       <label>Color scheme name</label>
-      <default>BreezeLight</default>
+      <default>SolusDark</default>
     </entry>
   </group>
   <group name="KDE">
     <entry name="singleClick" key="SingleClick" type="Bool">
       <label>Single click to open files</label>
-      <default>true</default>
+      <default>false</default>
     </entry>
     <entry name="lookAndFeelPackage" key="LookAndFeelPackage" type="String">
       <label>Global Look and Feel package</label>
-      <default>org.kde.breeze.desktop</default>
+      <default>solusdark.desktop</default>
     </entry>
     <entry name="defaultLightLookAndFeel" key="DefaultLightLookAndFeel" type="String">
       <label>Global Look and Feel package, alternate</label>
-      <default>org.kde.breeze.desktop</default>
+      <default>soluslight.desktop</default>
     </entry>
     <entry name="defaultDarkLookAndFeel" key="DefaultDarkLookAndFeel" type="String">
       <label>Global Look and Feel package, alternate</label>
-      <default>org.kde.breezedark.desktop</default>
+      <default>solusdark.desktop</default>
     </entry>
     <entry name="animationDurationFactor" key="AnimationDurationFactor" type="Double">
       <label>Animation speed</label>
-- 
2.40.1

